pyproject.toml

```diff
@@ -31,22 +33,7 @@ select = [
     "B011",
     "B013",
     "B014",
-    "C400",
```

Comment:
@logan-markewich you can see the heart of this PR: using full groups instead of calling out individual rules